recipes/gromacs_plumed/2020/build.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+# gromacs > 2021 requires a non default OSX version
+if [ "$(uname)" = 'Darwin' ] ; then
+    export MACOSX_DEPLOYMENT_TARGET=10.13 
+fi
+
+
+mkdir build
+cd build
+
+## See INSTALL of gromacs distro
+for ARCH in SSE2 AVX_256 AVX2_256; do
+  cmake_args=(
+    -DSHARED_LIBS_DEFAULT=ON
+    -DBUILD_SHARED_LIBS=ON
+    -DGMX_PREFER_STATIC_LIBS=NO
+    -DGMX_BUILD_OWN_FFTW=OFF
+    -DGMX_DEFAULT_SUFFIX=ON
+    # Tests are not currently run, so do not download them
+    # -DREGRESSIONTEST_DOWNLOAD=ON
+    -DCMAKE_PREFIX_PATH="${PREFIX}"
+    -DGMX_INSTALL_PREFIX="${PREFIX}"
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}"
+    -DGMX_SIMD="${ARCH}"
+    -DCMAKE_INSTALL_BINDIR="bin.${ARCH}"
+    -DCMAKE_INSTALL_LIBDIR="lib.${ARCH}"
+    -DGMX_VERSION_STRING_OF_FORK="bioconda"
```

Comment:
Should be bioconda-plumed so users know what produced their log files.

Reply:
> > We're failing on downloading this file: https://ftp.gromacs.org/pub/gromacs/gromacs-2020.6.tar.gz
> > It's an intermittant error, but more off than on at the moment. Will try bumping the build again tomorrow.
> 
> I'm getting it also at #31508

I wonder if we should try downloading from github instead? (i.e. from https://github.com/gromacs/gromacs/archive/refs/tags/v2021.4.tar.gz)

Reply:
Ahh, yes - I'll change that now.